packages/x-charts/src/themeAugmentation/themeAugmentation.spec.ts

```diff
@@ -188,5 +188,53 @@ createTheme({
       //   constent: { color: 'red' },
       // },
     },
+    MuiPieArc: {
+      defaultProps: {
+        id: 'toto',
+        // @ts-expect-error invalid MuiMarkElement prop
+        someRandomProp: true,
+      },
+      styleOverrides: {
+        root: { backgroundColor: 'red' },
+        // @ts-expect-error invalid MuiMarkElement class key
+        constent: { color: 'red' },
+      },
+    },
+    MuiPieArcLabel: {
+      defaultProps: {
+        id: 'toto',
+        // @ts-expect-error invalid MuiMarkElement prop
+        someRandomProp: true,
+      },
+      styleOverrides: {
+        root: { backgroundColor: 'red' },
+        // @ts-expect-error invalid MuiMarkElement class key
+        constent: { color: 'red' },
+      },
+    },
+    MuiChartsReferenceLine: {
```

Comment:
This one doesn't work at all. 🙈 
There is no styled component with this name and no `useThemeProps` usage. 🤷